lib/Runtime/Base/ScriptContext.h

```diff
@@ -52,6 +52,8 @@ class SRCINFO
     Js::ModuleID moduleID;
     ULONG grfsi;
 
+    Js::Var serializedScriptBuffer; // in order to keep a reference to serialized Script Buffer
```

Comment:
Why ifdef or "all the way down to" -- I see it's just a matter of doing it at https://github.com/obastemur/ChakraCore/blob/a2ab359471b48c7c5dade5eae4c0a2934f6bfce9/lib/Jsrt/Jsrt.cpp#L3261 vs. https://github.com/obastemur/ChakraCore/blob/a2ab359471b48c7c5dade5eae4c0a2934f6bfce9/lib/Jsrt/Jsrt.cpp#L3233 passing one more parameter to JsrtSourceHolder constructor.